Adaptors/Amqp/src/PushQueueStorage.cs

```diff
@@ -108,16 +110,22 @@ public class PushQueueStorage : QueueStorage, IPushQueueStorage
                                   $"{partitionId}###SenderLink{whichQueue}",
                                   $"{partitionId}###q{whichQueue}");
 
-      await Task.WhenAll(messages.Select(id => sender.SendAsync(new Message(Encoding.UTF8.GetBytes(id))
-                                                                {
-                                                                  Header = new Header
-                                                                           {
-                                                                             Priority = (byte)internalPriority,
-                                                                           },
-                                                                  Properties = new Properties(),
-                                                                })))
+      await Task.WhenAll(messages.Select(msgData => sender.SendAsync(new Message(Encoding.UTF8.GetBytes(JsonSerializer.Serialize(new
```

Comment:
As we return the taskId only, I would only send the taskId in.

Reply:
We can bypass them based on our needs.

Reply:
Tests are not working because of that. During the pull, we need to get the task id. You changed the serialization but you did not change the deserialization part !